exporter/awscloudwatchlogsexporter/exporter.go

```diff
@@ -66,7 +65,10 @@ func newCwLogsPusher(expConfig *Config, params component.ExporterCreateSettings)
 		return nil, err
 	}
 
-	expConfig.Validate()
+	err = expConfig.Validate()
+	if err != nil {
+		return nil, err
+	}
```

Comment:
This can go away entirely, validation is already done by the service so this will never fail.

Reply:
The "service" you mention is `cwlogs.NewClient` fucntion?

Reply:
No, it's the one on opentelemetry-collector. The call happens here https://github.com/open-telemetry/opentelemetry-collector/blob/c56d20e9e0afccc43179da12f60e24e5729caca9/service/config_provider.go#L117, which validates the configuration of all components https://github.com/open-telemetry/opentelemetry-collector/blob/c56d20e9e0afccc43179da12f60e24e5729caca9/config/moved_config.go#L53

Reply:
Yes, this validation should be able to be removed.  It seems overly defensive given that the service hosts will already have validated it.